src/convert.jl

```diff
@@ -986,3 +986,28 @@ function convert(::Type{Zonotope},
     Z2 = translate(Z1, vector(am), share=true)
     return Z2
 end
+
+"""
+    convert(::Type{<:HParallelotope}, Z::AbstractZonotope)
+
+Converts a zonotopic set of order 1 into a parallelotope in constraint
+representation.
+
+### Input
+
+- `HParallelotope` -- type used for dispatch
+- `Z`        -- zonotopic set
+
+### Output
+
+A parallelotope in constraint representation.
+"""
+function convert(::Type{<:HParallelotope}, Z::AbstractZonotope)
+    @assert order(Z) == 1 "cannot convert a zonotope that is not of order 1 to"*
+                            " a parallelotope"
+    n = Int(2order(Z)*dim(Z))
+    D = reduce(hcat, [constraints_list(Z)[i].a for i in 1:2:n])'
+    c = [constraints_list(Z)[i].b for i in 1:2:n-1]
+    append!(c, [constraints_list(Z)[i].b for i in 2:2:n])
```

Comment:
Computing the constraints is costly, so it should be done only once.
```suggestion
    constraints = constraints_list(Z)
    D = reduce(hcat, [constraints[i].a for i in 1:2:n])'
    c = [constraints[i].b for i in 1:2:n-1]
    append!(c, [constraints[i].b for i in 2:2:n])
```